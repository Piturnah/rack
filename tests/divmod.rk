10 5 / print
10 5 % print
10 divmod 4 print print

