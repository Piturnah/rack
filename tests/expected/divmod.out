----STDOUT----
2
0
2
2

----STDERR----
